tests/suites/cloud/tests/multicontainer/index.js

```diff
@@ -144,61 +67,20 @@ module.exports = {
         // set device service variable for frontend service
         await this.cloud.balena.models.device.serviceVar.set(
             this.balena.uuid,
-            services.current_services.frontend[0].service_id,
+            services.current_services.containerA[0].service_id,
             key,
             value
           );
 
         // Check to see if variable is present in front end service
         await this.utils.waitUntil(async () => {
           test.comment("Checking to see if variables are visible...");
-          let env = await this.cloud.executeCommandInContainer(`env`, `frontend`, this.balena.uuid)
+          let env = await this.cloud.executeCommandInContainer(`env`, `containerA`, this.balena.uuid)
 
           return env.includes(`${key}=${value}\n`);
         }, false, 60, 5 * 1000);
         test.ok(true, `Should service env var in service it was set for`);
       },
     },
-    {
-      title: "Move device back to original app",
```

Comment:
Reading this now, why did we even have this test? You might know the bigger picture

Reply:
@vipulgupta2048 I think it was just because that what was in testlodge

Reply:
Right on right on